benches/association_search_benchmark.rs

```diff
@@ -0,0 +1,52 @@
+use std::{collections::HashSet, path::PathBuf};
+
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use semsimian::{Predicate, RustSemsimian, TermID};
+
+fn criterion_benchmark(c: &mut Criterion) {
+    let mut db_path = PathBuf::new();
+    if let Some(home) = std::env::var_os("HOME") {
+        db_path.push(home);
+        db_path.push(".data/oaklib/phenio.db");
```

Comment:
Is this expecting the phenio db to already be present?

Reply:
That's probably fine for benchmark purposes though

Reply:
Yes. Since benchmarking will be run by only us (developers), it makes an assumption our `pystow` resource exists where it generally is supposed to.